credentials/development/gen-development-pai-test-vectors.sh

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env bash
+
+#
+# Copyright (c) 2022 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Script that can be used to generate Matter development device attestation test vectors
+# for testing purposes.
+# The script expects the name of the resulting source file containing the test vectors.
+#
+# Usage example when the script is run from the CHIP SDK root directory:
+#     ./credentials/development/gen-development-attestation-test-vectors.sh src/credentials/examples/ExamplePAI
+#
+# This command will generate the following C-Style files:
+#     src/credentials/examples/ExamplePAI.cpp
+#     src/credentials/examples/ExamplePAI.h
```

Comment:
Suggest not modifying this at all. We already have too many "example" PAI/PAA/DAC that actually confuse developers as to what they have to do. Let's work towards reducing the number of "test"/"dev" DAC PKI items over time and avoid touching this in this PR. Let's keep this PR related to just the trust store